ganga/GangaCore/Lib/Localhost/LocalHostExec_batch.py.template

```diff
@@ -37,17 +37,18 @@ sys.stdout=open(join(outdir,'__syslog__'),'w')
 sys.stderr=sys.stdout
 
 print(f'Running with a batch size of {batch_size}')
-try:
-    pool = multiprocessing.Pool(batch_size)
-    pool.map(mp_worker, scriptnames_g)
-finally:
-    pool.close()
-    pool.join()
+if __name__ == '__main__':
+    try:
+        pool = multiprocessing.Pool(batch_size)
+        pool.map(mp_worker, scriptnames_g)
+    finally:
+        pool.close()
+        pool.join()
```

Comment:
This looks really odd. The template code is executed through a `subprocess.run` so it should the if statement should always be true. And it case it wasn't, then the code would do nothing at all? So what is the purpose of this change?

Reply:
I wonder if your problems are due to the shebang at the start of the script. Do you get some kind of mixed environment when starting the process as `#!/usr/bin/env python3`?